code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -845,7 +879,7 @@
 	..()
 
 /datum/reagent/medicine/strange_reagent/on_mob_life(mob/living/carbon/M)
-	M.adjustBruteLoss(0.5*REM, 0)
+	M.adjustInternalLoss(0.5*REM, 0)
```

Comment:
b2i

Reply:
likewise with above, probably not a bad idea though